lms/resources/_js_config/__init__.py

```diff
@@ -495,10 +495,6 @@ def _hypothesis_client(self) -> Dict[str, Any]:
         # We cache this property (@functools.lru_cache()) so that it's
         # mutable. You can do self._hypothesis_client["foo"] = "bar" and the
         # mutation will be preserved.
-
-        if not self._application_instance.provisioning:
```

Comment:
These are not a huge burden but they complicate refactors.

Reply:
We'll fail early while getting the application instance, this code can't be reached with provisioning off.